src/linear_algebra/multilinear/basic.lean

```diff
@@ -284,6 +285,42 @@ def comp_linear_map (g : multilinear_map R M₁' M₂) (f : Π i, M₁ i →ₗ[
   g.comp_linear_map f m = g (λ i, f i (m i)) :=
 rfl
 
+/-- Composing a multilinear map twice with a linear map in each argument. -/
```

Comment:
This docstring (and all the others) feel weird, because they don't actually describe the lemma, just its LHS. I'd be inclined to just remove all the docstrings except `comp_linear_equiv_eq_zero_iff`, as the statements are trivial.